GitCommands/Git/GitItemStatus.cs

```diff
@@ -159,10 +159,16 @@ public bool IsStatusOnly
         /// </remarks>
         public bool IsRangeDiff
         {
-            get => _flags.HasFlag(Flags.IsRangeDiff);
+            get => HasFlag(Flags.IsRangeDiff);
             set => SetFlag(value, Flags.IsRangeDiff);
         }
 
+        private bool HasFlag(Flags flags)
+        {
+            // NOTE Enum.HasFlag boxes its argument
+            return (flags & _flags) == flags;
+        }
```

Comment:
BTW: The name of the enum `Flags` is misleading. It should be in singular.
I can read this better:
```suggestion
        private bool HasFlag(Flags flag)
        {
            // NOTE Enum.HasFlag boxes its argument
            return (_flags & flag) == flag;
        }
```

Reply:
The logic works equally well with multiple flags. The type is a flags enum, and has a plural name.

Reply:
> The logic works equally well with multiple flags.

This was clear to me. But `Enum.HasFlag` uses only singular. And nothing else is used here.

Better readability (as in other recent PRs):
```suggestion
        private bool HasFlag(Flags flags)
        {
            // NOTE Enum.HasFlag boxes its argument
            return (_flags & flags) == flags;
        }
```